src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java

```diff
@@ -382,17 +411,18 @@ private MinigameTask cloneCrosswordpuzzle(final MinigameTask minigameTask, final
                     minigameTask.getIndex()
                 );
             } catch (final FeignException e) {
-                if (!errorMessages.contains("crosswordpuzzle-backend not present")) {
-                    log.debug(CLONE_ERROR_MESSAGE, e);
-                    errorMessages.add("crosswordpuzzle-backend not present");
-                    return new MinigameTask(Minigame.CROSSWORDPUZZLE, "", null, minigameTask.getIndex());
-                }
+                log.debug(CLONE_ERROR_MESSAGE, e);
```

Comment:
In an exception, debug level? Also the backend name should be mentioned in the log.